unit-tests/live/fw/test-fw-errors.py

```diff
@@ -157,8 +157,56 @@ def get_error_summary(self):
         log.i(f"Streaming completed - Duration: {elapsed_time:.1f}s, Total frames: {frame_count}")
         
     finally:
-        pipe.stop()
-        log.d("Pipeline stopped")
+        # Stop pipeline first (if running) and then attempt to cleanly stop/close all sensors
```

Comment:
Please remove all of this section, the GenAI you used is wrong, pipeline stop close all the sensors itself